client/state/jetpack-connection-health/schema.js

```diff
@@ -0,0 +1,42 @@
+export const requestError = {
+	type: [ 'string', 'null' ],
+};
+
+export const isLoading = {
+	type: 'boolean',
+};
+
+export const lastRequestTime = {
+	type: 'number',
+};
+
+export const connectionHealth = {
+	type: 'object',
+	properties: {
+		is_healthy: {
+			type: 'boolean',
+		},
+		jetpack_connection_problem: {
+			type: 'boolean',
+		},
+		error: {
+			type: 'string',
+		},
+	},
+};
+export const jetpacConnectionHealth = {
```

Comment:
There is a typo here:

```suggestion
export const jetpackConnectionHealth = {
```